### PR TITLE
lists: Update list management page

### PR DESCRIPTION
**Pull request by greezybacon**
This patch revamps the list management page. The new item manager is paginated, which supports list items over 327 items on most web servers. It also adds support for CSV import and allows for mass actions on the items include delete, disable, and enable.

##### Simplified, tabular, paginated view of list items, with mass actions
The table shows the first four properties defined in the properties tab

Simplified, tabular, paginated view of list items, with mass actions

##### Value, Abbreviation, and Properties editable in popup dialog
Value and Abbreviation editable in popup dialog

Properties editable in popup dialog

##### List properties now includes visibility settings
List properties now includes visibility settings

##### List items can be imported via CSV file too
List items can be imported via CSV file too

### CODE REVIEW

**Comment by greezybacon** [comment]
Fixes #1055

**Comment by Chefkeks** [comment]
Great to finally (watched the corresponding branch :D ) see some screenshots for this!
Really good work!

**Comment by protich** [comment]
@greezybacon - perhaps I missed it, but do we have a rebase of this PR for develop branch?

File: include/ajax.forms.php [hunk 1]
```diff
@@ -96,29 +96,228 @@ function deleteAnswer($entry_id, $field_id) {
...
+        $item_form = $this->_getListItemEditForm($_POST, $item);
+
+        if ($valid = $item_form->isValid()) {
+            // Update basic information
+            $basic = $item_form->getClean();
+            $item->extra = $basic['extra'];
+            $item->value = $basic['value'];
+
+            if ($_item = DynamicListItem::lookup(array('value'=>$item->value)))
+                if ($_item && $_item->id != $item->id)
```
Comment (protich) [inline include/ajax.forms.php]: ``$item`` is undefined here

Comment (greezybacon) [inline include/ajax.forms.php]: Isn't it defined on line `153`?

**Pull request merged**